packages/storybook-blocks/src/styles/tokens/colors.scss

```diff
@@ -26,7 +26,7 @@
   --sb-text-color-fixed-dark: #323338;
   --sb-inverted-color-background: #323338;
   --sb-disabled-background-color: #ecedf5;
-  --sb-primary-background-hover-color: #dcdfec;
+  --sb-primary-background-hover-color: rgba(103, 104, 121, 0.1);
```

Comment:
I think we should try to use Vibe for the storybook styling

Reply:
I guess the intention was to avoid circular dependency with style when style had its own storybook

Reply:
worth opening a tech debt task for it? @talkor

Reply:
Yes sure will open